BlueChip/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "BlueChip",
+    "assets": [
+      {
+        "description": "The native staking token of BlueChip.",
+        "denom_units": [
+          {
+            "denom": "ubluechip",
+            "exponent": 0
+          },
+          {
+            "denom": "ebl",
```

Comment:
This is likely incorrect, left over from a template assetlist json file. I assume this is meant to be 'bcp'